.github/workflows/twister_tests.yml

```diff
@@ -10,6 +10,7 @@ on:
     - v*-branch
     paths:
     - 'scripts/pylib/twister/**'
+    - 'scripts/pylib/build_helpers/**'
```

Comment:
I believe this would fit better as part of the Scripts tests workflow, separate from twister.

Reply:
I have now separated the domains tests to `scripts/tests/build_helpers/test_domains.py` and created a separate workflow for them, called `.github/workflows/pylib_tests.yml`.

Reply:
Works for me, thanks!